### DISCUSSION TITLE
编译报错，启动是正常的

### DISCUSSION THREAD

**Discussion by MarkTang3611** [alex8088/electron-vite#671] [Q&A]
PS E:\learn\xy-print> pnpm build:win

> xy-print@1.0.0 build:win E:\learn\xy-print
> npm run build && electron-builder --win

> xy-print@1.0.0 build
> npm run typecheck && electron-vite build

> xy-print@1.0.0 typecheck
> npm run typecheck:node && npm run typecheck:web

> xy-print@1.0.0 typecheck:node
> tsc --noEmit -p tsconfig.node.json --composite false

> xy-print@1.0.0 typecheck:web
> vue-tsc --noEmit -p tsconfig.web.json --composite false

E:\learn\xy-print\node_modules\.pnpm\vue-tsc@2.1.10_typescript@5.7.2\node_modules\vue-tsc\index.js:34
            throw err;
            ^
Search string not found: "/supportedTSExtensions = .*(?=;)/"
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.11.0

**Comment by yuhou-cl** [comment]
Same problem, how to solve?

**Reply by zzg** [reply]
vue-tsc 版本切换成固定的 2.0.29
typescript 版本切换成固定的 5.6.2"


```
  "devDependencies": {
    "vue-tsc": "2.0.29",
    "typescript": "5.6.2",
    "vite": "^5.4.10",
    "@vitejs/plugin-vue": "^5.1.4"
  }

```

然后重新安装依赖
